frontend/src/api/client.js

```diff
@@ -22,4 +25,16 @@ client.interceptors.response.use(function (response) {
     return Promise.reject(error)
 })
 
+// 500 Internal Server Errors
+client.interceptors.response.use(function (response) {
```

Comment:
Personally would have put this all into the existing interceptor to save every request having to pass through an extra Promise. But this is okay.

Reply:
Yeah, that's fair, I did it more for cleanliness of code/reading. Will happily merge them in the future.